skylib/k8s.bzl

```diff
@@ -241,6 +241,17 @@ def k8s_deploy(
             image_digest_tag = image_digest_tag,
             tags = tags,
         )
+        image_names = {}
+        if images:
+            for img_name, target_name in images.items():
+                trg = target_name.replace("//", "").split(":")
+                repo = trg[0] + "/" + trg[1]
+                image_names = {
```

Comment:
Doesn't this overwrite `image_names` for each image in `images.items()`?

Reply:
yup, you're right, when I was doing this I forgot to go back to this and figure out how to set a list of dictionaries as an argument, bazel doesn't seem to have that type available https://bazel.build/rules/lib/attr, I think I can maneuver around it though